docs/api/server/CLI-Commands.md

```diff
@@ -0,0 +1,217 @@
+[👈 Return to Overview](../API.md)
+
+# CLI Commands
+
+`one-app` Server consists of [NPM Script Commands](https://docs.npmjs.com/misc/scripts) that determine its runtime configuration and behaviors. 
+
+```sh
+npm run <one-app-npm-command> [--] [--some-flag]
+# e.g. npm run test:unit -- --watch
+```
+
+We describe each command and a description of its usage below.
+
+**Contents**
+* Install Commands
+  * [`preinstall`](#install-commands)
+  * [`postinstall`](#install-commands)
+* Development Commands
+  * [`serve-module`](#test-commands)
+  * [`drop-module`](#test-commands)
+  * [`set-middleware`](#test-commands)
+  * [`set-dev-endpoints`](#test-commands)
+* Start Commands
+  * [`start`](#start-commands)
+  * [`start:watch`](#start-commands)
+  * [`prestart:prod-sample`](#start-commands)
+  * [`start:prod-sample`](#start-commands)
+* Build Commands
+  * [`build`](#build-commands)
+  * [`build:server`](#build-commands)
+  * [`build:bundle`](#build-commands)
+  * [`build:artifacts:statics`](#build-commands)
+  * [`build:prod-sample`](#build-commands)
+  * [`build:sample-modules`](#build-commands)
+* Clean Commands
+  * [`clean`](#clean-commands)
+  * [`clean:build`](#clean-commands)
+  * [`clean:test`](#clean-commands)
+* Test Commands
+  * [`test`](#test-commands)
+  * [`test:unit`](#test-commands)
+  * [`test:lint`](#test-commands)
+  * [`pretest:integration`](#test-commands)
+  * [`test:integration`](#test-commands)
+  * [`test:danger`](#test-commands)
+  * [`test:git-history`](#test-commands)
+  * [`test:lockfile`](#test-commands)
+
+## Install Commands
+
+| Command       | Description                                                                                    |
+|---------------|------------------------------------------------------------------------------------------------|
+| `preinstall`  | Runs `check-engines` package to ensure a compatible Node version is used before `npm install`. |
+| `postinstall` | Runs a [`build`](#build-commands) after `npm install`.                                         |
+
+## Development Commands
+
+> 💬 These commands only work in Development
```

Comment:
Technically these commands all are only for development right?

Reply:
What do you mean? I have this callout under Development Commands already in there:

> 💬 These commands only work in Development

Reply:
what about rephrasing to "These commands are for Development only"